src/nicolai_munsterhjelm_ex/ex04/myrand.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nicolai Munsterhjelm'
+__email__ = 'nicolai.munsterhjelm@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
```

Comment:
Please put a docstring in all important methods that take input from or returns something to the user. Documentation for the class should be put as a docstring in the __init__ function.